common/src/main/java/com/microsoft/identity/common/internal/migration/TokenCacheItemMigrationAdapter.java

```diff
@@ -496,25 +496,14 @@ public static ADALTokenCacheItem findFrt(
      */
     @NonNull
     public static String getScopesForTokenRequest(@NonNull final String v1Resource) {
-        String scopes = getScopeFromResource(v1Resource);
+        String scopes = OAuthUtil.getScopeFromResource(v1Resource);
```

Comment:
Since this is an Microsoft STS specific function.  Doesn't make sense in the context of any other IDP... probably shouldn't go in an OAuthUtil class.  Could go on the MicrosoftSTSOAuth2Strategy class.